Source/BSN.IpTables.Cli/command.ps1

```diff
@@ -0,0 +1,49 @@
+# Show current version
```

Comment:
What is this script?

Please rename to good name

Please add good comments

Reply:
This file is removed.